### PR TITLE
[CUB-2527] Base config for typescripts

### PR DESCRIPTION
**Pull request by ignatiusreza**
The tsconfig is taken from Laboperator tsconfig.. if there're any differences with the tsconfig used in other repo, I'm happy to discuss..

### CODE REVIEW

**Review by tamagosante — APPROVED** [review]

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (ghost) [inline tsconfig.json:1]: Won't adding "allowJs": true be something helpful for making the transition to ts? I guess how it is now, its either typescript or js with no in-between.

**Review by ghost — APPROVED** [review]
In general, LGTM :100: Just left a question on the tsconfig file

File: tsconfig.json (hunk 1 above)
Comment (ignatiusreza) [reply tsconfig.json:1]: we're using typescript via babel, which already understand javascript, so `allowJs` is not necessary to allow ts and js to exists side by side..

Comment (ghost) [reply tsconfig.json:1]: Ohh that makes sense, thanks!

**Pull request merged**